test/unit/policy_group_test.rb

```diff
@@ -34,4 +37,24 @@ class PolicyGroupTest < ActiveSupport::TestCase
     Whitehall::PublishingApi.expects(:publish_async).with(policy_group).once
     policy_group.publish_to_publishing_api
   end
+
+  test "#published_policies should return the all policies" do
+    policy_group = create(:policy_group)
+    rummager_has_new_policies_for_every_type
+
+    all_policy_titles = [
```

Comment:
This is now duplicated in two sets of tests. Worth extracting to a helper to remove the duplication?